### DISCUSSION TITLE
Custom DnsNameResolver

### DISCUSSION THREAD

**Discussion by ErikOwsiak** [netty/netty#12072] [General]
How or can default DnsNameResolver class be replaced with a custom class?

**Comment by normanmaurer** [comment]
@ErikOwsiak You would implement your own `AddressResolver` / `AddressResolverGroup`

**Reply by ErikOwsiak** [reply]
@normanmaurer tnx for feedback ... are there good examples that you can recommend? we basically need to fallback to config loaded IP for each domain if dns lookup fails for any reason. 
so this seems to be all over the net:
Caused by: io.netty.resolver.dns.DnsNameResolverTimeoutException: [/10.10.11.11:53] query timed out after 5000 milliseconds (no stack trace available)
we need a workaround for this issue
